contributions/catalog/requirements.txt

```diff
@@ -1,4 +1,6 @@
 Flask==1.1.1
```

Comment:
```suggestion
Flask==1.1.1
werkzeug==2.0.2
```
add additional requirement

Reply:
Hi, @bingzhang, it looks like this suggestion was resolved, but the `werkzeug` change has not been added to the catalog/requirements.txt. Please fix this. I'm unresolving this comment. Thanks.

Reply:
Thanks, Bing. I will mark this resolved.